test/dynamo/test_exc.py

```diff
@@ -287,14 +286,14 @@ def fn(x, shape):
 Model:
   ==> L['shape'][0]: 1
   ==> L['shape'][1]: 1
-  ==> L['shape'][2]: 2
+  ==> L['shape'][2]: 0
```

Comment:
what happened here?

Reply:
I mean, Z3 is welcome to pick whatever model it wants lol. I guess removing some of the constraints changed what it picked. You can see that 0 is valid because we only have >= 0 bounds on these variables, rather than >= 2.